### PR TITLE
Pin postgres

### PR DESCRIPTION
**Pull request by TylerSheffels**
@dannyroberts

### CODE REVIEW

File: ansible/roles/postgresql/tasks/main.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 - name: Install PostgreSQL & dependencies
   apt: name={{ item }} state=present
   with_items:
-    - postgresql
+    - postgresql={{ postgresql_version }}*
```
Comment (snopoke) [inline ansible/roles/postgresql/tasks/main.yml]: This diff seems out of date with master:
```
- name: Install PostgreSQL & dependencies
  apt: name={{ item }} state=present
  with_items:
    - postgresql
    - postgresql-{{ postgresql_version }}
    - libpq-dev
    - python-psycopg2
  when:
    pg_install == True
```

**Comment by dannyroberts** [comment]
FYI @TylerSheffels due to my delinquency in timely review, this is now not mergeable

**Comment by TylerSheffels** [comment]
@dannyroberts No sweat, will resolve merge conflicts

**Comment by dannyroberts** [comment]
bumpalump

**Comment by TylerSheffels** [comment]
Looks like master got updated to pin the PG version in another PR

**Pull request closed without merging**